Wikipedia/Code/WMFImageController.swift

```diff
@@ -219,7 +219,11 @@ public class WMFImageController : NSObject {
     public func typedDiskDataForImageWithURL(url: NSURL?) -> WMFTypedImageData {
         if let url = url {
             let path = imageManager.imageCache.defaultCachePathForKey(cacheKeyForURL(url))
-            let mimeType: String? = NSFileManager.defaultManager().wmf_valueForExtendedFileAttributeNamed(WMFExtendedFileAttributeNameMIMEType, forFileAtPath: path)
+            var mimeType: String? = NSFileManager.defaultManager().wmf_valueForExtendedFileAttributeNamed(WMFExtendedFileAttributeNameMIMEType, forFileAtPath: path)
+            if mimeType == nil {
+                let fileURL = NSURL(fileURLWithPath: path, isDirectory: false)
+                mimeType = fileURL.pathExtension?.wmf_asMIMEType()
+            }
```

Comment:
So the downside is that some images will be broken in the browser, right? Because we needed to do this for SVGs I believe, correct?